packages/ui/src/components/va-layout/VaLayout.vue

```diff
@@ -117,9 +117,11 @@ export default defineComponent({
   width: 100vw;
 
   &__area {
-    @include va-scroll();
+    @include va-scroll(var(--va-primary));
 
     &--content {
+      overflow-x: visible;
```

Comment:
Spotted a regression:

# Before:
![localhost_3000_ui-elements_scroll-container (5)](https://github.com/epicmaxco/vuestic-ui/assets/5394573/5f64b267-e9c2-4052-836a-d23477d4f59c)



#After:
![localhost_3000_ui-elements_scroll-container (2)](https://github.com/epicmaxco/vuestic-ui/assets/5394573/53ea8efa-2422-416a-9dc8-a94f35909e27)

You can read more about this here (#3496), but it's critical for visual testing that we can screenshot the whole page.

Reply:
https://github.com/epicmaxco/vuestic-ui/pull/3889/files/4c3431edf2dcd09261e09dd1ded00e4e7078a0a1#diff-4dae880321fed108b33e744a82a182de9b33acc723adeb35c919f123a93c2b74R116

The height property in `.va-layout` class force the content to stay inside div. However, without overflow property in the `&--content` section, the anchor overscroll would not work as detailed in the issue. Setting the overflow property seems to cause the regression issue. 

To support both overscroll and fix the regression, it would be ideal to fix the layout's `height` prop. Any suggestions here @asvae ?

Reply:
I fixed full page screenshot, but anchor scroll seems to be broken. I think we just need to add offset to scroll equal to navbar height.